docs/_blog/2024_q3_update.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "2024 Q3 Update: What Have We Been Up To?"
+description: "2024 Q3 Update on the Bitcoin Dev Kit Project"
+authors:
+    - thunderbiscuit
+date: "2024-11-01"
+tags: ["BDK", "project"]
+draft: false
+---
+
+### Q3 2024: What Have We Been Up To?
```

Comment:
Nothing wrong with having this line of text `### Q3 2024: What Have We Been Up To?` at all, but when I look at the [preview](https://deploy-preview-202--awesome-golick-685c88.netlify.app/blog/_2024-q3-update/) just because of the way the UI formats it ends in relation to that divider, it ends up looking a bit cramped

<img width="1088" alt="Screenshot 2024-11-07 at 1 25 23 PM" src="https://github.com/user-attachments/assets/59715f33-7b7a-4cf0-9e5c-91d0ac22b228">

Now we dont need to fix the UI divider stuff right this moment obv, but maybe my suggestion would be to remove that line of text `### Q3 2024: What Have We Been Up To?` since in the [Q2 update](https://bitcoindevkit.org/blog/_2024-q2-update/) we don't have this line which helps the UI look *slightly* better

<img width="1094" alt="Screenshot 2024-11-07 at 1 25 42 PM" src="https://github.com/user-attachments/assets/25c144ba-628f-4e53-9c86-3e4d5e641b46">

Reply:
Oh yes of course.